src/sentry/issues/ingest.py

```diff
@@ -113,7 +113,7 @@ def _create_issue_kwargs(
         "first_release": release,
         "data": materialize_metadata(occurrence, event),
     }
-    kwargs["data"]["last_received"] = event.datetime
+    kwargs["data"]["last_received"] = json.datetime_to_str(event.datetime)
```

Comment:
Could we convert this to a timestamp instead?

Reply:
actually I would prefer `str` here as it means no special processing is needed to migrate it from pickle -- just loading and storing

Reply:
Sorry, should have explained my reasoning.

This field is also stored on error issues as a float: https://github.com/getsentry/sentry/blob/0db44d6460577e5a9ae25ed59bae630f218089bf/src/sentry/event_manager.py#L2333. We missed that due to an oversight, but it makes sense to me to keep the format consistent.

But if it's a huge pain to handle the special processing I'm ok with str.